installation/release-notes-main/release-notes-9-0-0.adoc

```diff
@@ -0,0 +1,77 @@
+= BAMOE, 9.0.0, release notes
+include::../../partials/attributes.adoc[]
+
+Highlights, new features, removed components, and differences between {PRODUCT} {VERSION_OLD} and {VERSION}:
+
+== Components
+
+* Runtimes
+** Decision Service
+** Observability
+*** Auditability (Metrics)
+*** Tracing via Kafka
+
+== New Components
+
+* IBM Business Automation Manager Open Editions Canvas 
+** Accelerators
+*** Work with individual assets or full projects
+*** Simple projects visualization
+*** Easy branch management
+*** Share
+**** download full projects
+**** individual decision files (DMN)
+**** svgs
+**** or obtain an embeddable version of your decision
+** Improved Collaboration
+*** Support for GitHub, GitHub Enterprise, Bitbucket Cloud
+*** Seamless git integration
+**** connect 
+**** clone (public and private repositories) 
+**** pull 
+**** push
+**** commit with custom messages
+** Decision Test capabilities
+*** Automatically generated forms 
+*** Export for 
+** One-click deploy (Dev/test purposes)
+*** OpenShift environments
+*** Kubernetes
+** Run on containerized environments with new product image 
+* IBM Business Automation Manager Open Editions Developer Tools
+** Available on the marketplace
+* Runtime support for DMN 1.4
+* Accelerators
+* Upstream-first 
+** Upstream certified versions available through new IBM Bills Of Material (dependency management)
+* Cloud
+** Helm Charts
+
+== Deprecated
+
+* Red Hat Business Automation VSCode Bundle
+
+== Removed Components
+* OpenShift Business Automation Operator
+* Business Central
+* KIE Server
+* Smart Router
+* Process Instance Migration Tool
+* Dashbuilder Runtime
+* Process Migration Service
+* OpenShift Images
```

Comment:
this makes little sense to me, this list is about v8.. while v9 is a completely different architecture and runtime stack.

Reply:
k, I didn't see the diffs in that way... but if the content was already there.. ok.

@LightGuard can you confirm? :)

Reply:
Please see my comment above for this whole file.

Reply:
This PR is moving the release notes to a different folder structure. I propose they be moved to a different PR.

Reply:
Yes, existing content and is in previous release 9.0.0